cmd/release-controller/sync.go

```diff
@@ -475,6 +478,22 @@ func (c *Controller) syncAccepted(release *Release) error {
 	return nil
 }
 
+func (c *Controller) syncTests(release *Release) error {
+	testTags := findTagReferencesByPhase(release, releasePhaseAccepted, releasePhaseRejected)
+	if glog.V(4) && len(testTags) > 0 {
+		glog.Infof("release=%s validation=%v", release.Config.Name, tagNames(testTags))
+	}
+	for _, tag := range testTags {
+		if changed, _ := c.updateUpgradeJobs(release, tag); changed {
```

Comment:
This should really be called `ensureAdditionalTests` - it's job is to ensure that at any point in time, the jobs are being run.  if one of them fails that's fine, except you *must* return an error in that case so that you get re-queued to try again.  So it's better to return right away if there is an error on a tag.

Reply:
`ensureAdditionalValidationJobs` might be a good name.  It's not the same as verification (ready + verification -> accepted|rejected)